### DISCUSSION TITLE
Optimism Sepolia faucet backed by Proof of Machinehood

### DISCUSSION THREAD

**Discussion by shawnonchain** [ethereum-optimism/developers#76] [General]
### Discussion Topic

Hey all! Just wanted to share some updates to support devs building on OP! 

We have launched L2Faucet for Optimism - where you can receive Sepolia ETH through attestation - without the need of social verifications, sign ups or task lists. Do try it out and let me have your feedback!

Try it out here: https://www.l2faucet.com/optimism

**Comment by sbvegan** [comment]
Nice! That was a very smooth experience: https://sepolia-optimism.etherscan.io/tx/0x584d56bf6cdcbcc2e1593099bd7610f10433ad3ae5929ffcc99690ea4e08d707
